lib/ui/recommendation.js

```diff
@@ -19,16 +19,21 @@ function Recommendation(opts) {
   this.io = opts.io;
   this.objectUtils = opts.objectUtils;
   this.RecommendationRow = opts.RecommendationRow;
+  this.transforms = {
+    tld: opts.tldTransform,
+    wikipedia: opts.wikipediaTransform,
+    movie: opts.movieTransform
+  };
 
   this.el = null;
-  this.timeout = null;
+  this.hideTimeout = null;
```

Comment:
Or maybe get rid of that change. Wasn't there a moment when I tried using two timeouts, hence, needed to distinguish between them?